app/assets/stylesheets/govuk_admin_template/_bootstrap_colour_overrides.scss

```diff
@@ -1,8 +1,47 @@
+$black: #000;
+
 /*
   Bootstrap 3.2 made gray-light darker for accessibility and contrast:
   https://github.com/twbs/bootstrap/releases/tag/v3.2.0
 
   This assumed gray-light was on a light background. We aren't always
   using it in that way. Reset to the original value #999.
 */
-$gray-light: lighten(#000, 60%);
+
+$gray-light: lighten($black, 60%);
+
+/*
+  Updates a bunch of copy that used to be gray to black for things like labels.
+*/
+$text-color: $black;
+
+/*
+  Updates the input to black so it contrasts with the light gray background.
+*/
+$input-color: $black;
+
+/*
+  Updates muted text to gray so it contrasts with the light gray background.
+*/
+$text-muted: #444;
```

Comment:
Not a blocker but maybe it would be easier to read if you had:

```
$gray-dark: #444444;

...

$text-muted: $gray-dark;

...

$breadcrumb-active-color: $gray-dark;
```

Reply:
Cool have sorted!